isobus/src/isobus_virtual_terminal_client.cpp

```diff
@@ -361,705 +340,520 @@ namespace isobus
+			                                         static_cast<std::uint8_t>(fillPatternObjectID >> 8),
+			                                         0xFF };
+		return queue_command(buffer, CANIdentifier::CANPriority::PriorityLowest7);
 	}
 
-	bool VirtualTerminalClient::send_draw_line(std::uint16_t objectID, std::int16_t xOffset, std::int16_t yOffset) const
+	bool VirtualTerminalClient::send_change_active_mask(std::uint16_t workingSetObjectID, std::uint16_t newActiveMaskObjectID)
 	{
-		const std::array<std::uint8_t, CAN_DATA_LENGTH> buffer = { static_cast<std::uint8_t>(Function::GraphicsContextCommand),
-			                                                         static_cast<std::uint8_t>(objectID & 0xFF),
-			                                                         static_cast<std::uint8_t>(objectID >> 8),
-			                                                         static_cast<std::uint8_t>(GraphicsContextSubCommandID::DrawLine),
-			                                                         static_cast<std::uint8_t>(xOffset & 0xFF),
-			                                                         static_cast<std::uint8_t>(xOffset >> 8),
-			                                                         static_cast<std::uint8_t>(yOffset & 0xFF),
-			                                                         static_cast<std::uint8_t>(yOffset >> 8) };
-		return CANNetworkManager::CANNetwork.send_can_message(static_cast<std::uint32_t>(CANLibParameterGroupNumber::ECUtoVirtualTerminal),
-		                                                      buffer.data(),
-		                                                      CAN_DATA_LENGTH,
-		                                                      myControlFunction,
-		                                                      partnerControlFunction,
-		                                                      CANIdentifier::PriorityLowest7);
+		const std::vector<std::uint8_t> buffer = { static_cast<std::uint8_t>(Function::ChangeActiveMaskCommand),
+			                                         static_cast<std::uint8_t>(workingSetObjectID & 0xFF),
+			                                         static_cast<std::uint8_t>(workingSetObjectID >> 8),
+			                                         static_cast<std::uint8_t>(newActiveMaskObjectID & 0xFF),
+			                                         static_cast<std::uint8_t>(newActiveMaskObjectID >> 8),
+			                                         0xFF,
+			                                         0xFF,
+			                                         0xFF };
+		return queue_command(buffer, CANIdentifier::CANPriority::PriorityLowest7, true);
 	}
 
-	bool VirtualTerminalClient::send_draw_rectangle(std::uint16_t objectID, std::uint16_t width, std::uint16_t height) const
+	bool VirtualTerminalClient::send_change_softkey_mask(MaskType type, std::uint16_t dataOrAlarmMaskObjectID, std::uint16_t newSoftKeyMaskObjectID)
```

Comment:
Hmm the GitHub diff is a little nasty, I recommend using the GitLens extension in vscode to view the changes: 
![image](https://github.com/Open-Agriculture/AgIsoStack-plus-plus/assets/29043784/62606d8d-b2e0-4283-b5f4-1419c3a69dcb)